### ISSUE TITLE
Cannot play music

### ISSUE THREAD

**Issue by Nathan1123** [gravesjohnr/AlexaNotificationCurl#7]
Just finished setting it up. Ran the command:
`./alexa.sh "play music"`
The response (from my computer) announced the music it was going to play, but no music actually played. I even tried logging into Amazon's web player. No dice.
I then tried asking my real Alexa to play music, and was surprised to hear "Amazon music is already being streamed from another device". So it did make the request to stream music, but didn't actually play it.
I also tried switching the default music app from Amazon to Spotify, and got the same results. Funny enough, after sending the request to play on Spotify using the "alexa.sh" script, I was still able to play Amazon music on the real Alexa. Apparently there is no confliction if its different services. 

Bottom line is, did you try using this method to play music at all? If so, what am I doing wrong? If not, I would recommend that is a huge disadvantage, being it is a large percentage of what Alexa's basic functionality is. 
Nathan

**Comment by gravesjohnr** [comment]
Nathan,

I haven't tried to stream using this method.  You can look at the 
response to see if it sends some type of streaming url or mp3u type 
file.  If so, you could play it using something like VLC.

This script expects a complete audio file to be returned and then 
played.  Not streamed.

-John


------------------------------------------------------------------------

*From:* Nathan1123
*Sent:* Thursday, May 18, 2017 9:37AM
*To:* Gravesjohnr/alexanotificationcurl
*Cc:* Subscribed
*Subject:* [gravesjohnr/AlexaNotificationCurl] Cannot play music (#7)
